### ISSUE TITLE
Webflow CMS

### ISSUE THREAD

**Issue by eclectrickink** [alvarotrigo/fullPage.js#3236]
Hi Alvaro, 

Are you familiar with implementing fullpage.js within webflow (with CMS dynamically loaded data)? I've been able to implement it half way but for some reason, it skips over the last CMS content section as well as the footer.

The is a link to an isolated page (duplicated version of the homepage
www.radley.tv/home-test

I also made another duplicate of the page with just the very basic code
https://www.radley.tv/home-test-base-code
`<script type="text/javascript" src="https://cdn.rawgit.com/alvarotrigo/fullPage.js/master/jquery.fullPage.js"></script>
<script>
$(document).ready(function() {
	$('#fullpage').fullpage({
  	css3: true,
		scrollBar: true,
		scrollOverflow: true,
		paddingTop: '0px',
		paddingBottom: '0px',

		//events
		onLeave: function(index, nextIndex, direction){},
		afterLoad: function(anchorLink, index){},
		afterRender: function(){},
		afterResize: function(){},
		afterResponsive: function(isResponsive){},
		afterSlideLoad: function(anchorLink, index, slideAnchor, slideIndex){},
		onSlideLeave: function(anchorLink, index, slideIndex, direction, nextSlideIndex){}
	});
});
</script>
`
I have no idea why it's fighting me - hopefully you can provide some insight. FYI I'm not an expert in code like you so it would be helpful if you can slightly dumb it down for me. 

Thanks!

**Comment by alvarotrigo** [comment]
> The is a link to an isolated page (duplicated version of the homepage

That's not an isolated reproduction :)
You should read my article regarding [how to ask coding questions to get answers](https://alvarotrigo.com/blog/how-not-suck-at-asking-coding-questions/).

>  it skips over the last CMS content section as well as the footer.

Regarding the footer, that's because the footer be treated also as a section. And it should be inside the fullpage.js wrapper. 
Anything that is outside the fullpage.js wrapper will have to be with position:fixed or otherwise won't ever be accessible (when using autoScrolling: true, as it's your case).

**Comment by eclectrickink** [comment]
I apologize for not understanding the isolated reproduction. I don't really know how to do that in webflow and your article doesn't say anything in relation to webflow so I'm kind of lost. 

I think I understand about the footer but I'm still confused as to why it skips over the last CMS div that is wrapped in the fullpage.js wrapper.

**Comment by alvarotrigo** [comment]
If you need my support services, please [contact me](https://alvarotrigo.com/#contact) and I'll provide you more details about it.

Here in the forum I only provide support to issues containing isolated reproductions to make sure the issue is within fullpage.js and not outside it. I won't check any website which is not isolated.
